pkg/controllers/deprovisioning/suite_test.go

```diff
@@ -1128,6 +1128,7 @@ func ExpectNewMachinesDeleted(ctx context.Context, c client.Client, wg *sync.Wai
 
 	wg.Add(1)
 	go func() {
+		GinkgoHelper()
```

Comment:
Not actually sure here but do you need to do two GinkgoHelpers()? One at the top of this function `ExpectNewMachinesDeleted` and the other the goroutine? I think the stack trace won't print correctly if this failed (to be clear I don't think this works correctly today either from looking at it, but this is a good time to fix it 😄 )

Reply:
ok i just thought since the goroutines are decoupled from the original function they might need separate attention since they work separately

Reply:
ya similar thought here